config/routes.rb

```diff
@@ -10,6 +10,7 @@
   namespace :api do
     scope :v1 do
       get "ping" => "ping#index"
+      get "validate_token" => "validate_token#index"
```

Comment:
I also didn't think to hard about this, but I'd like to consider whether we can make this more resource-y rather than RPC-y.

Reply:
Another thing: I prefer to normalize on URLs with hyphens not underscores, because:

- slug normalization uses hyphens, which means that when we parameterize things in the API, if we use underscores for URL segments, then we get a mix of underscores and hyphens which is less than ideal
- google uses hyphens to separate words, so if we ever want things to be searchable, hyphens tend to be better in terms of understandability for the search engine